docs/artifactory/config/help.go

```diff
@@ -13,6 +15,12 @@ const Arguments string = `	server ID
 	show
 		Shows the stored configuration.
 		In case this argument is followed by a configured server ID, then only this server's configurations is shown.
+	
+	import
+		Imports an artifactory server configuration from server token.
+	
+	export
+		Creates an Artifactory server configuration token.
```

Comment:
Creates an Artifactory server configuration token. -->
Creates an Artifactory server configuration token. The generated token can be imported by the "jfrog rt c import [Server token]" command.